perma_web/perma/forms.py

```diff
@@ -505,7 +505,7 @@ def clean_csv_file(self):
         for row in reader:
             row_count += 1
             email = row.get('email')
-            email = email.strip() if email else None
+            email = email.strip().lower() if email else None
```

Comment:
Note that this will result in us actually saving the lowercase version of the email in `raw_email`, rather than preserving the submitted capitalization. Not that big a deal; a bug similar to the long-standing LIL-2921. If you can think of a way to just case to lowercase before filtering, that might be best, but if you cannot, we can fix that some other time (maybe along with LIL-2921).

Reply:
Great, thanks for doing this!